marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -163,15 +177,41 @@ data TxMintRow = TxMintRow
     , _txMintRowRedeemerData    :: !C.ScriptData
     , _txMintRowRedeemerHash    :: !(C.Hash C.ScriptData)
     }
-    deriving (Eq, Ord, Show, Generic, SQL.FromRow, SQL.ToRow)
+    deriving (Eq, Ord, Show, Generic)
 
 makeLenses 'TxMintRow
 
+instance SQL.FromRow TxMintRow where
```

Comment:
It looks like something changed so that now you could use generic deriving again?